src/frontend/komponenter/ManuellJournalføring/DokumentVelger.tsx

```diff
@@ -0,0 +1,132 @@
+import { IDokumentInfo, IJournalpost } from '@navikt/familie-typer';
+import Panel from 'nav-frontend-paneler';
+import React, { useState } from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import { DokumentIkon } from '../../ikoner/DokumentIkon';
+import { useManuellJournalføring } from '../../context/ManuellJournalføringContext';
+
+const DokumentPanel = styled(Panel)`
+    margin-top: 20px;
+    width: 27rem;
+    height: 100%;
+    border: ${props => `${props.theme.borderWidth} solid ${props.theme.borderColor}`};
+    &:hover {
+        border-color: ${props => `${props.theme.hoverBorderColor}`};
+    }
+`;
+
+const DokumentInfoStripeContainer = styled.div`
+    display: flex;
+    justify-content: left;
```

Comment:
Just curious about this one. What is it defaulting to? I though it would place itself left by default, but it is not?

Reply:
Yes, this has been removed:)